requirements.txt

```diff
@@ -1,6 +1,9 @@
 asgiref==3.6.0
 Django==4.2
 django-cors-headers==4.1.0
+django-rest-framework==0.1.0
```

Comment:
Looks like there are two "django rest framework" packages. Is this one needed?

Reply:
uhhh strange and kind of scary! This is the one we need:
https://pypi.org/project/djangorestframework/

This other one looks sus af: https://pypi.org/project/django-rest-framework/

Have I been hacked?